futurex_openedx_extensions/dashboard/docs_src.py

```diff
@@ -1819,4 +1819,58 @@ def get_optional_parameter(path: str) -> Any:
             remove=[200, 404],
         ),
     },
+
+    'CourseAssetsManagementView.create': {
+        'summary': 'Add new asset to the tenant',
+        'description': 'Add new asset to the tenant. \n `Note:` If an asset with the same slug already exists, the '
+        'existing record will be updated with the new file instead of creating a duplicate entry.',
```

Comment:
```suggestion
        'existing record will be updated with the new file instead of creating a duplicate entry. The old file will **not** be deleted from the storage.',
```